gradle.properties

```diff
@@ -2,4 +2,4 @@ version=0.11.0-SNAPSHOT
 kestraVersion=0.11.+
 micronautVersion=3.9.3
 lombokVersion=1.18.28
-awsVersion=2.20.53
```

Comment:
same downgrade here

Reply:
2.20.53 => 2.20.117 seems like an ugrade to me